app/__init__.py

```diff
@@ -8,6 +8,7 @@
 from config import Config
 
 app = Flask(__name__)
+app.secret_key = '34983459843jf4985435kdsjcf3489fjc3j34c'
```

Comment:
@msusicky k čemu to je?

Reply:
Kvuli session, bez toho je nemuzes pouzivat. Casem bychom to asi meli vytahnout do konfigu, ale neni to nic kritickeho.